accounts/models.py

```diff
@@ -1,3 +1,36 @@
 from django.db import models
+from django.db import models
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+
+
+class Profile(models.Model):
+    USER_TYPE_CHOICES = [
+        ('buyer', 'Buyer'),
+        ('seller', 'Seller'),
+    ]
```

Comment:
I don't think a type is needed. Like on OLX, a user should be able to buy as well as sell stuff.

Reply:
ok sir I will resolve it.